srml/balances/src/lib.rs

```diff
@@ -302,7 +302,11 @@ impl<T: Trait> Module<T> {
 		if would_create && value < Self::existential_deposit() {
 			return Err("value too low to create account");
 		}
-		Self::ensure_account_can_withdraw(transactor, value, WithdrawReason::Transfer, new_from_balance)?;
+		Self::ensure_account_can_withdraw(
+			transactor,
+			WithdrawReason::Transfer | WithdrawReason::TransactionPayment,
```

Comment:
This is not a transaction payment. Transaction payments happen specifically for inclusion of a transaction in the chain.